test/ui-test/src/screens/StatusCommunityScreen.py

```diff
@@ -125,3 +128,28 @@ def edit_community(self, new_community_name: str, new_community_description: str
 
     def go_back_to_community(self):
         click_obj_by_name(CommunitySettingsComponents.BACK_TO_COMMUNITY_BUTTON.value)
+
+    def delete_current_community_channel(self):
+        wait_for_banner_to_disappear()
+
+        click_obj_by_name(CommunityScreenComponents.CHAT_MORE_OPTIONS_BUTTON.value)
+        click_obj_by_name(CommunityScreenComponents.DELETE_CHANNEL_MENU_ITEM.value)
+        click_obj_by_name(CommunityScreenComponents.DELETE_CHANNEL_CONFIRMATION_DIALOG_DELETE_BUTTON.value)
+
+    def attempts_to_delete_current_community_channel_but_cancels(self):
+        wait_for_banner_to_disappear()
+
+        click_obj_by_name(CommunityScreenComponents.CHAT_MORE_OPTIONS_BUTTON.value)
+        click_obj_by_name(CommunityScreenComponents.DELETE_CHANNEL_MENU_ITEM.value)
+        press_esc(CommunityScreenComponents.DELETE_CHANNEL_CONFIRMATION_DIALOG_DELETE_BUTTON.value)
+
+    def check_channel_count(self, count_to_check: int):
+        chatListObj = get_obj(CommunityScreenComponents.NOT_CATEGORIZED_CHAT_LIST.value)
+        # Squish doesn't follow the type hints when parsing gherkin values
+        verify_equals(chatListObj.statusChatListItems.count, int(count_to_check))
+
+# Wait for the banner to disappear otherwise the click might land badly
+def wait_for_banner_to_disappear():
```

Comment:
Just an FYI, I improved this function is my PR https://github.com/status-im/status-desktop/pull/6916/files#diff-fc8026bde3f79315654aa05293a5575f08c0af154fb22c1dcbf79d6b664a04bcR71
So whoever merges last will need to rebase and use that function istead.

Reply:
Let the race begin :))